eos/b-decays/b-to-dstar-l-nu_TEST.cc

```diff
@@ -186,31 +132,48 @@ class BToDstarLeptonNeutrinoTest :
                 p3["mass::b(MSbar)"]              = +4.18;
                 // mc(mc)
                 p3["mass::c"]                     = +1.275;
+                // mu mode
                 p3["b->cmunumu::Re{cVL}"]         = +1.0;
-                p3["b->cmunumu::Im{cVL}"]         = -1.0;
+                p3["b->cmunumu::Im{cVL}"]         = -2.0;
                 p3["b->cmunumu::Re{cVR}"]         = +2.0;
                 p3["b->cmunumu::Im{cVR}"]         = -2.0;
                 p3["b->cmunumu::Re{cSL}"]         = +3.0;
                 p3["b->cmunumu::Im{cSL}"]         = -3.0;
                 p3["b->cmunumu::Re{cSR}"]         = +4.0;
                 p3["b->cmunumu::Im{cSR}"]         = -4.0;
-                // switch on the tensor op. contr. here
                 p3["b->cmunumu::Re{cT}"]          = +5.0;
                 p3["b->cmunumu::Im{cT}"]          = -5.0;
+                // tau mode
+                p3["b->ctaunutau::Re{cVL}"]       = +1.0;
```

Comment:
As last thing: Could you put here in tau-NP Wilson coefficients different
numerical values then you are using above for mu-NP Wilson coefficients,
because in the way it is now, you might would not been able to catch the
mistake with the missing "opt_l" that you had in the differential R_Dstar

Reply:
Yes, i see what you mean. I will do this too, I will let you know when I push my newer changes for a last look before merging to the base branch.

Reply:
done!